src/core/QueryManager.ts

```diff
@@ -1220,29 +1235,38 @@ export class QueryManager<TStore> {
       setTimeout(() => concast.cancel(reason));
     });
 
-    // A Concast<T> can be created either from an Iterable<Observable<T>>
-    // or from a PromiseLike<Iterable<Observable<T>>>, where T in this
-    // case is ApolloQueryResult<TData>.
-    const concast = new Concast(
-      // If the query has @export(as: ...) directives, then we need to
-      // process those directives asynchronously. When there are no
-      // @export directives (the common case), we deliberately avoid
-      // wrapping the result of this.fetchQueryByPolicy in a Promise,
-      // since the timing of result delivery is (unfortunately) important
-      // for backwards compatibility. TODO This code could be simpler if
-      // we deprecated and removed LocalState.
-      this.transform(normalized.query).hasClientExports
-        ? this.localState.addExportedVariables(
-          normalized.query,
-          normalized.variables,
-          normalized.context,
-        ).then(fromVariables)
-        : fromVariables(normalized.variables!)
-    );
+    let concast: Concast<ApolloQueryResult<TData>>, containsDataFromLink: boolean;
```

Comment:
```suggestion
    let concast: Concast<ApolloQueryResult<TData>>, 
        containsDataFromLink: boolean;
```

I know this seems petty, but I honestly searched for a while before I realized this variable was defined in the same line as `concast`. For readability, could we put this on its own line to make it easier to see where this variable is created?

Reply:
Not petty, perfectly valid complaint :)